test/e2e/upgrade/upgrade.go

```diff
@@ -188,6 +188,13 @@ var _ = g.Describe("[sig-arch][Feature:ClusterUpgrade]", func() {
 						clusterUpgrade(f, client, dynamicClient, config, upgCtx.Versions[i]),
 						fmt.Sprintf("during upgrade to %s", upgCtx.Versions[i].NodeImage))
 				}
+				err = wait.PollImmediate(1*time.Second, 30*time.Second, func() (bool, error) {
+					if err := checkUpgradeability(client); err != nil {
```

Comment:
I don't think polling `checkUpgradeability` will improve the situation as it doesn't check for any statuses that would report the deamonset as not fully operating. I approved the changes with this check in the other PR because I inadvertently thought that it was this test, but this one doesn't check for cluster upgradeability.

I initially went with a time.Sleep because I couldn't find a status that would be easy to poll, but maybe we could find something.